integ-test/src/test/resources/correctness/expressions/functions.txt

```diff
@@ -10,6 +10,19 @@ ceil(-1)
 ceil(0.0)
 ceil(0.4999)
 ceil(abs(1))
+power(2, 2)
+power(2, -2)
+power(2.1, 2)
+power(2, -2.1)
+power(abs(2), 2)
+sign(0)
+sign(-1)
+sign(1)
+sign(abs(1))
+sqrt(0)
+sqrt(1)
+sqrt(1.1)
+sqrt(abs(1))
```

Comment:
Any reason why not test `mod` here? I can see the support in H2: http://www.h2database.com/html/functions.html#mod

Reply:
currently the schema does not match with H2

Reply:
Is name or type different?

Reply:
@dai-chen It's name. Currently all tests of functions have different name in schema with H2. For example the query `SELECT SIGN(1.1)` has a column name of `SIGN(1.1)` while H2 has the result as the column name, which looks like:
```
"schema": [{
    "name": "1",
    "type": "INTEGER"
}]
```

There are still a couple of functions that have different behavior compared to SQLite, while has the same behavior with H2, like the `sign` function above but schema name is different. These cases are also added to manual test rather than the comparison test cases for now.

Reply:
I see. Sorry I forgot that! We should be able to overcome this once we support column alias.